config/prod/prow/config_knative.yaml

```diff
@@ -277,6 +277,11 @@ presubmits:
     always-run: true
     command:
     - ./test/presubmit-integration-tests-latest-release.sh
+  knative/client-pkg:
+  - build-tests: true
+  - unit-tests: true
+  - integration-tests: true
```

Comment:
@rhuss I'd say that we don't necessarily need integration tests for client-pkg. Wdyt? We should be fine with build and unit tests.

Reply:
yes, there is not really something we could test integration wise. Maybe the watch functionality, but I would add integration tests when we would need it the first time. let's start small.

Reply:
Or maybe we'll keep it blank and come up with something meaningful? :)